google/cloud/security/inventory/pipelines/load_projects_iam_policies_pipeline.py

```diff
@@ -99,7 +99,7 @@ def _retrieve(self):
         """
         # Get the projects for which we will retrieve the IAM policies.
         try:
-            project_numbers = self.dao.select_project_numbers(
+            project_numbers = self.dao.get_project_numbers(
```

Comment:
Yes. get_project_numbers() has been moved to project_dao and I also pass the project_dao to the constructor for LoadProjectsIamPoliciesPipeline. I guess it's not so clear here, sorry :|